bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/BlueZBluetoothDevice.java

```diff
@@ -50,7 +49,9 @@
 
     private final Logger logger = LoggerFactory.getLogger(BlueZBluetoothDevice.class);
 
-    private final ScheduledExecutorService scheduler = ThreadPoolManager.getScheduledPool("bluetooth");
+    private final ScheduledExecutorService scheduler = ThreadPoolManager.getScheduledPool("bluetooth.bluez");
```

Comment:
We usually use a simple string (potentially camel case) for pool names, but no dot notation.
I'd think "bluetooth" is just fine - if there are other bluetooth bundles making use of it, even better as that's what a pool is for.

Reply:
done, I changed it back